src/pyodide/internal/workers.py

```diff
@@ -682,6 +719,138 @@ async def text(self) -> str:
         return await self.js_object.text()
 
 
+def serialize_to_python(obj: "JsProxy"):
```

Comment:
Don't love the name -- if the wire format is JavaScript, then isn't this deserializing? What about `python_from_rpc` and `python_to_rpc`?